pkg/restapi/vc/operation/operations.go

```diff
@@ -270,6 +284,22 @@ func (o *Operation) storeVCHandler(rw http.ResponseWriter, req *http.Request) {
 	}
 }
 
+// This function expects the vcID that's passed in to be a URL with a "/" and then a UUID concatenated to the end.
+func convertVCIDToEDVCompatibleFormat(vcID string) (string, error) {
+	vcIDParts := strings.Split(vcID, "/")
+
+	uuidFromVCID := vcIDParts[len(vcIDParts)-1]
+
+	parsedUUID, err := uuid.Parse(uuidFromVCID)
+	if err != nil {
+		return "", fmt.Errorf(fmt.Sprintf("%s: %s", invalidUUIDErrMsg, err.Error()))
```

Comment:
fmt.Sprintf is redundant here... Errorf() does the same thing

Reply:
Oops, good catch. Fixed.